R/helpers.R

```diff
@@ -1,10 +1,15 @@
 #' Interpret input text as Markdown-formatted text
 #'
+#' Markdown! It's a wonderful thing. We can use it in certain places (e.g.,
+#' footnotes, source notes, the table title, etc.) and expect it to render to
+#' HTML as Markdown does. There is the [html()] helper that allows you to ferry
+#' in HTML but this function `md()`... it's almost like a two-for-one deal (you
+#' get to use Markdown plus any HTML fragments *at the same time*).
```

Comment:
Probably should add a note that it's not safe to put untrusted user input in here (like database values that can be influenced by a potentially malicious user--even a username!--or literally anything from Shiny `input`) as it'd then be possible to inject JavaScript into the end result document.

Also, when you support RTF/LaTeX, these docs will need to change to talk about output formats generically instead of HTML.

Reply:
I going to make this separate advisory happen in a separate PR. I believe it's important and I want to get the language right to ensure there's no misunderstanding.